Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1beta3/defaults_test.go

```diff
@@ -470,23 +470,32 @@ func areSecurityContextAndContainerEqual(c *versioned.Container) (bool, []string
 	return equal, issues
 }
 
+func newBool(b bool) *bool {
+	ptr := new(bool)
+	ptr = &b
+	return ptr
+}
+
 func TestDefaultSecurityContextConstraints(t *testing.T) {
 	tests := map[string]struct {
-		scc              *versioned.SecurityContextConstraints
-		expectedFSGroup  versioned.FSGroupStrategyType
-		expectedSupGroup versioned.SupplementalGroupsStrategyType
+		scc                   *versioned.SecurityContextConstraints
+		expectedFSGroup       versioned.FSGroupStrategyType
+		expectedSupGroup      versioned.SupplementalGroupsStrategyType
+		expectedAllowEmptyDir bool
 	}{
 		"shouldn't default": {
 			scc: &versioned.SecurityContextConstraints{
 				FSGroup: versioned.FSGroupStrategyOptions{
 					Type: versioned.FSGroupStrategyMustRunAs,
 				},
+				AllowEmptyDirVolumePlugin: newBool(true),
 				SupplementalGroups: versioned.SupplementalGroupsStrategyOptions{
 					Type: versioned.SupplementalGroupsStrategyMustRunAs,
 				},
 			},
-			expectedFSGroup:  versioned.FSGroupStrategyMustRunAs,
-			expectedSupGroup: versioned.SupplementalGroupsStrategyMustRunAs,
+			expectedFSGroup:       versioned.FSGroupStrategyMustRunAs,
+			expectedSupGroup:      versioned.SupplementalGroupsStrategyMustRunAs,
+			expectedAllowEmptyDir: true,
```

Comment:
This would then be identical to "preserve AllowEmptyDirVolumePlugin set to false" at the end, the only case missing was set to true and check true. Probably shouldn't be messing around with this test at all, it's named generically but I don't think it's actually meant for every property possible. Instead I should probably just add another test at the end for set true check true. Set nothing and expect true would be covered by all pre-existing tests.